assembler/templates/home.html

```diff
@@ -0,0 +1,7 @@
+<!-- wp:template-part {"slug":"header"} /-->
```

Comment:
We don't need to include a Home template, because the Assembler flow will always add it (see Creatio 2).

Reply:
This theme (and the existing Creatio 2) won't be discoverable outside the Assembler flow, so this template will never be possibly usable.

We had this convo before, see: p1665497100719849/1665152933.113869-slack-CRWCHQGUB.

Reply:
In the future it may be available elsewhere (perhaps leveraged by WooCommerce even). Would the theme + Assembler on dotCom function properly with home.html existing in the theme files, even if it's not necessarily used?

Reply:
I see. I believe it should work -- let's 🚢 and try!